boards/rpi/config.cfg

```diff
@@ -5,6 +5,8 @@
 # use the boards pin names directly in your printer.cfg instead. 
 # Look up the pins you need in the boards pinout diagram.
 
-[mcu rpi]
+# Kept for compatibility; Migrated to boards/host-computer/config.cfg
+
+[mcu host_computer]
```

Comment:
this should still be [mcu rpi], or user configs referencing rpi pins will break.

Reply:
added this change too